### DISCUSSION TITLE
NodeRed Gmail kann keine Emails mehr versenden

### DISCUSSION THREAD

**Discussion by Ulf349** [pixelit-project/PixelIt#178] [General]
Hallo, 

Google hat den unsicheren Emailversand von Drittprogrammen entfernt. Gibt es eine Möglichkeit trotzdem weitere Gmail Emails aus NodeRed zu versenden?

**Comment by foorschtbar** [comment]
Mit der PixelIt hat das aber nichts zu tun oder übersehe ich da was?

Ansonsten weiß ich nicht wie du vorher die Mails verschickt hast, aber das Stichwort heißt SMTP und sollte auch weiterhin funktionieren. Auf die schnelle habe ich das gefunden: https://support.google.com/mail/answer/7126229
